example_project_bzl_4/third_party/readline/BUILD.bazel.tmpl

```diff
@@ -0,0 +1,21 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "all",
+    srcs = glob([
+        "lib/*.a",
+        "lib/*.so*",
+    ]),
+    hdrs = glob([
+        "include/**/*.h",
+        "include/**/*.hpp",
+    ]),
+    includes = [
+        "include",
+    ],
+    deps = [
+        "@ncurses.lib//:all",
```

Comment:
Technically, this isn't needed as readline pulls the dependency itself. I'm wondering whether it will be the case with static linking, too.